### PR TITLE
Added initial README with useful starting point links.

### PR DESCRIPTION
**Pull request by bwplotka**
Fixes https://github.com/cncf/sig-observability/issues/8

cc @halcyondude @RichiH

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,72 @@
...
-# Structure
+## Mission Statement
 
-## Chairs, in alphabetical order
+The SIG Observability focuses on topics pertaining to the observation of cloud native workloads. Additionally, it produces
+supporting material and best practices for end-users and provides guidance and coordination for CNCF projects working within the SIG’s scope.
 
-* Matt Young, [Everquote](https://www.everquote.com/)
-* Richard "RichiH" Hartmann, [Grafana](https://grafana.com/), Prometheus [team](https://prometheus.io/governance/#team-members), [PromCon](https://promcon.io/) lead
+### What is in the Scope?
```
Comment (RichiH) [inline README.md:10]:
```suggestion
### Scope
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,72 @@
...
 
-## Tech Lead
+* Foster, review and grow the ecosystem of observability related projects, users, and maintainers in the open source, cloud-native scope.
```
Comment (RichiH) [inline README.md:12]:
```suggestion
* Foster, review and grow the ecosystem of observability related projects, users, and maintainers in open source, cloud-native technology.
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,72 @@
...
+* Identify and report gaps in the CNCF's project portfolio on topics of observability to the TOC and the wider CNCF community.
+* Collect, curate, champion, and disseminate patterns and current best practices related to the observation of cloud-native systems that are effective and actionable. 
+Educate and inform users with unbiased, accurate, and pertinent information. Educate and help other CNCF projects in regarding observability techniques and practices available within the CNCF.
```
Comment (RichiH) [inline README.md:15]:
```suggestion
Educate and inform users with unbiased, accurate, and pertinent information. Educate and help other CNCF projects regarding observability techniques and best current practices available within the CNCF.
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,72 @@
...
+* Provide and maintain a vendor-neutral venue for relevant thought validation, discussion, and project feedback.
+* Provide a ladder for community members to become involved with the technical oversight of projects within the SIG's scope in an open, transparent, and inclusive way.
+
+## Starting Point
```
Comment (RichiH) [inline README.md:19]:
```suggestion
## Officially sanctioned information channels
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,72 @@
-# sig-observability
+# CNCF SIG Observability 🔭⚙️
 
-Special Interest Group for Observability under the umbrella of [CNCF](https://www.cncf.io/) 🔭⚙️
+Special Interest Group for Observability under the umbrella of [CNCF](https://www.cncf.io/).
 
-# Structure
+## Mission Statement
```
Comment (RichiH) [inline README.md:5]:
```suggestion
## Mission statement
```

File: README.md [hunk 6]
```diff
@@ -1,12 +1,72 @@
...
+
+1. Ensure there is no similar issue already in [GitHub Issues](https://github.com/cncf/sig-observability/issues)
+2. [Add new issue](https://github.com/cncf/sig-observability/issues/new) with explanation.
+3. If the issue is large and you would like to have quicker feedback, add the issue number as a topic for next SIG Observability meeting on our meeting notes.
+
+### How Can I help?
+
+The best way to start is to look [at issues in the backlog on our Kanban Board](https://github.com/cncf/sig-observability/projects/1#card-36042463).
+If you are happy to help in any of those, comment on the issue. Also do not hesitate to ask us on our communication channels or GitHub for more details if needed!
+
```
Comment (RichiH) [inline README.md:43]:
```suggestion
## How to get involved

The best way to get involved is hanging around our channels of communication, joining our calls, and getting a feeling for how we operate.

If you want to tackle specific tasks, it's best to look at our [Kanban Board](https://github.com/cncf/sig-observability/projects/1#card-36042463) and pick out what you like.

If you want to suggest a specific topic or action item, please do some basic research if this topic has already been tackled, or is currently being worked on, in some form. Good starting points are our [GitHub Issues](https://github.com/cncf/sig-observability/issues), reports, or meeting notes. If in doubt, asking on Slack is a good first step.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,72 @@
...
+| Matt Young          | `myoung@everquote.com` | `@Matt Young`      | [@halcyondude](https://github.com/halcyondude) | [Everquote](https://www.everquote.com/) |                                                                                                          |
+| Richard Hartmann    | `richardhartmann.de`   | `@RichiH`          | [@RichiH](https://github.com/RichiH)           | [Grafana](https://grafana.com/)         |  Prometheus [Team](https://prometheus.io/governance/#team-members); [PromCon](https://promcon.io/) Lead  |
+
+### Tech Lead
+
+| Name                | Email                  | CNCF Slack         | GitHub                                         | Company                                 | Open Source              |
+|---------------------|------------------------|--------------------|------------------------------------------------|-----------------------------------------|--------------------------|
```
Comment (RichiH) [inline README.md:62]: I don't think tables add much value here and would prefer to avoid them, but I don't feel strongly either way.

Comment (bwplotka) [reply README.md:52]: I really like them. Those really give very clean look and are very readable. The key is also to have contact details, (even though all of you have such detail in your GH profile).

I would vote to have that :hugs:

Comment (bwplotka) [reply README.md:52]: Also don't see harm. (:

**Comment by RichiH** [comment]
Thanks!

**Comment by bwplotka** [comment]
:tada:

**Pull request merged**